__test__/unit/notifications/eventListeners.test.ts

```diff
@@ -0,0 +1,22 @@
+import { TestEnvironment } from '../../support/environment/TestEnvironment';
+import EventHelper from '../../../src/shared/helpers/EventHelper';
+
+describe('Notification Events', () => {
+  beforeEach(async () => {
+    TestEnvironment.initialize();
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('Adding click listener fires internal EventHelper', async () => {
+    const stub = test.stub(
+      EventHelper,
+      'fireStoredNotificationClicks',
+      null,
```

Comment:
This is an optional argument. And I don't believe this function returns anything

Reply:
ah good catch, removed the `null`.